storage_service/static/js/project.js

```diff
@@ -1,5 +1,30 @@
 
-
 $(document).ready(function() {
-    $('.datatable').dataTable();
+    $('.datatable').dataTable({
+      // List of language strings from https://datatables.net/reference/option/language
+      oLanguage: {
```

Comment:
I wasn't familiar with the `oLanguage` setting, I've been using just `language`. Are the equivalent? I followed https://datatables.net/manual/i18n.